docs/examples/en/geometries/TextGeometry.html

```diff
@@ -25,7 +25,7 @@ <h2>Import</h2>
 		</p>
 
 		<code>
-			import { TextGeometry } from 'three/addons/geometries/TextGeometry.js';
+			import { TextGeometry } from 'three/examples/jsm/geometries/TextGeometry.js';
```

Comment:
We have changed the import path to `addons` on purpose. 

If you are using this import statement directly in the browser, an entry in your import map like in the examples is required.

Reply:
Am I supposed to have a `three/addons` folder by default? The docs suggest yes, the issues tracker suggests yes, but there's nothing there:

![image](https://github.com/mrdoob/three.js/assets/6835891/2c693790-3ddb-4860-8ef8-14b64ae405a4)

Reply:
`/addons` is just an alias to `examples/jsm`.

Reply:
Ah I see. Thanks for your reply @Mugen87. 

I looked for `examples/jsm` but didn't see it. I see nowthat it's actually called `examples/js` and indeed in contains the expected contents.

Reply:
> I see nowthat it's actually called examples/js and indeed in contains the expected contents.

Are you using an outdated repository? `examples/js` is actually deleted since `r148`.

Reply:
I sure am. I didn't realize I shouldn't be using `three.js` but instead `three`. I guess that'll teach me to freehand my install commands.